lib/commands/init/index.js

```diff
@@ -85,6 +86,9 @@ function initAlexaHostedSkill(rootPath, cmd, profile, cb) {
                             return cb(hooksErr);
                         }
                         Messenger.getInstance().info(`\n${skillName} successfully initialized.\n`);
+                        Messenger.getInstance().warn('If updating the skill package via the developer console, your git repo will be out of sync. '
```

Comment:
Any suggestions for a message here?

Basically people need to know if they create a hosted cli project, it doesn't stay synced with the UI dev console changes if they're making changes in both places

Reply:
you might want to sync up with a doc writer?